src/components/sections/link-item.tsx

```diff
@@ -56,3 +58,8 @@ export default function LinkItem({
     </TouchableOpacity>
   );
 }
+
+const useStyles = StyleSheet.createThemeHook(() => ({
```

Comment:
could this be part of `useSectionStyle`? Or should it just be specific of this particular section item?

Reply:
You tell me! It is probably not only this section item which may overflow, but moving `flex: 1` as a field of `sectionStyle` does not fix other section items automatically. 

Should I expand this PR to also fix other sections items?

Reply:
Hm. I actually think it might be extended. But let's not handle this right now.